mne_features/univariate.py

```diff
@@ -1097,6 +1160,9 @@ def compute_svd_entropy(data, tau=2, emb=10):
     return -np.sum(np.multiply(sv_norm, np.log2(sv_norm)), axis=-1)
 
 
+compute_svd_entropy.get_feature_names = _compute_generic_feat_names
```

Comment:
I'd rather do `if not hasattra(function, 'get_feature_names')` then use _compute_generic_feat_names. It avoids having to alter any possible function. Otherwise we need an OO design with a base class... which is a lot more work

Reply:
Agreed, yet as all univariate functions do not necessarily return a `(n_channels,)`-shaped output (for instance `compute_spect_slope`), it could be equally dangerous to use a generic function directly in `FeatureFunctionTransformer.fit()`. As I see it, there are two approaches given the current implementation:
- either define a function `_compute_[funcname]_feat_names` for every univariate function and assign it (`compute_[funcname].get_feature_names = _compute_[funcname]_feat_names`), but then it would simpler /clearer to be make it OO I guess)
- assign a generic list of names in `FeatureFunctionTransformer.transform()` if the output channel dim is equal to the one of the input (has to be done in the `transform()` in order to get the shape of the output) <= I implemented this one in 
b6158aa, it's a bit messy though